src/renderers/shared/event/EventPluginRegistry.js

```diff
@@ -265,6 +279,16 @@ var EventPluginRegistry = {
         delete registrationNameModules[registrationName];
       }
     }
+
+    if (__DEV__) {
+      var getPossibleRegistrationName = EventPluginRegistry
```

Comment:
style nit: Can you break after the equals sign instead of before the dot and indent only two spaces?